client/src/api/base_urls.tsx

```diff
@@ -1 +1 @@
-export const getWinesUrl = "http://localhost:4000/api/wines"
+export const getWinesUrl = (process.env.NODE_ENV == 'production' ? "https://wine-o-inventory.herokuapp.com/api/wines" : "http://localhost:4000/api/wines")
```

Comment:
I wonder if this would get complicated once you have more environments (`dev`, `qa`, `staging`, etc.). Do you have ideas on how to address that?